script/example.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python3
+import yaml
+import json
+from argparse import ArgumentParser
+
+def action_json(args):
+
+    with open('scripts/example_data.json') as json_file:
```

Comment:
Consider using a context manager for file operations to ensure that the file is properly closed after its contents are processed.
```suggestion
with open('scripts/example_data.json', 'r') as json_file:
```